net/tsaddr/tsaddr_test.go

```diff
@@ -89,3 +92,102 @@ func TestUnmapVia(t *testing.T) {
 		}
 	}
 }
+
+func TestIsExitNodeRoute(t *testing.T) {
+	tests := []struct {
+		pref netip.Prefix
+		want bool
+	}{
+		{
+			pref: AllIPv4(),
+			want: true,
+		},
+		{
+			pref: AllIPv6(),
+			want: true,
+		},
+		{
+			pref: netip.MustParsePrefix("1.1.1.1/0"),
+			want: false,
```

Comment:
This would be an example where a `.Bits() == 0` check will pass, and a full prefix case will fail.